include/groonga/cache.h

```diff
@@ -95,6 +95,18 @@ grn_cache_close(grn_ctx *ctx, grn_cache *cache);
 
 GRN_API grn_rc
 grn_cache_current_set(grn_ctx *ctx, grn_cache *cache);
+/**
+ * \brief Retrieve the current \ref grn_cache object used in the select command.
```

Comment:
Other commands such as `logical_select` use cache.

Reply:
[6dcde80](https://github.com/groonga/groonga/pull/2244/commits/6dcde800c224e859ac33db1c14aea53620dbfa73) : Thanks, I updated the content without specified use cases here.